examples/cluster-nodes-status.fsx

```diff
@@ -0,0 +1,140 @@
+#r "../src/Akkling.Cluster.Sharding/bin/Debug/Akkling.Persistence.dll"
+#r "../src/Akkling.Cluster.Sharding/bin/Debug/Akkling.Cluster.Sharding.dll"
+
+
+open Akka.Actor
+open Akka.Configuration
+open Akka.Cluster
+open Akka.Cluster.Tools.Singleton
+open Akka.Cluster.Sharding
+open Akka.Persistence
+
+open Akkling
+open Akkling.Persistence
+open Akkling.Cluster
+open Akkling.Cluster.Sharding
+open Wire
+
+let configWithPort port =
+    let config = Configuration.parse ("""
+        akka {
+            actor {
+              provider = "Akka.Cluster.ClusterActorRefProvider, Akka.Cluster"
+              serializers {
+                wire = "Akka.Serialization.WireSerializer, Akka.Serialization.Wire"
+              }
+              serialization-bindings {
+                "System.Object" = wire
+              }
+            }
+          remote {
+            helios.tcp {
+              public-hostname = "localhost"
+              hostname = "localhost"
+              port = """ + port.ToString() + """
+            }
+          }
+          cluster {
+            auto-down-unreachable-after = 5s
+            seed-nodes = [ "akka.tcp://cluster-system@localhost:5000/" ]
+          }
+          persistence {
+            journal.plugin = "akka.persistence.journal.inmem"
+            snapshot-store.plugin = "akka.persistence.snapshot-store.local"
+          }
+        }
+        """)
+    config.WithFallback(ClusterSingletonManager.DefaultConfig())
+
+type ClusterStatus = Get
+let clusterStatus = fun (ctx : Actor<_>) ->
+    let log = ctx.Log.Value
+    let cluster = Cluster.Get(ctx.System)
+    let rec loop nodes = actor {
+        let! (msg: obj) = ctx.Receive ()
+        match msg with
+        | LifecycleEvent e ->
+            match e with
+            | PreStart ->
+                cluster.Subscribe(ctx.Self, ClusterEvent.InitialStateAsEvents,
+                    [| typedefof<ClusterEvent.IMemberEvent>; typedefof<ClusterEvent.UnreachableMember> |])
+                log.Info (sprintf "Actor subscribed to Cluster status updates: %A" ctx.Self)
+            | PostStop ->
+                cluster.Unsubscribe(ctx.Self)
+                log.Info (sprintf "Actor unsubscribed from Cluster status updates: %A" ctx.Self)
+
+            | _ -> return Unhandled
+            return! loop nodes
+        | IMemberEvent e ->
+            match e with
+            | MemberJoined m ->
+                log.Info (sprintf "Node joined: %A" m)
+                return! loop (nodes |> Map.add (m.Address.ToString(), m.UniqueAddress.ToString()) m.Status)
+            | MemberUp m ->
+                log.Info (sprintf "Node up: %A" m)
+                return! loop (nodes |> Map.add (m.Address.ToString(), m.UniqueAddress.ToString()) m.Status)
+            | MemberLeft m ->
+                log.Info (sprintf "Node left: %A" m)
+                return! loop (nodes |> Map.add (m.Address.ToString(), m.UniqueAddress.ToString()) m.Status)
```

Comment:
I don't think `Map.add` is valid response on `MemberLeft` and `MemberExited`. You can pass just unchanged nodes here. 
Since this serves as example, recommended place for adding nodes is `MemberUp` and for removing them `MemberRemoved`.